lib/linkify.dart

```diff
@@ -67,7 +86,7 @@ class TextElement extends LinkifyElement {
   bool equals(other) => other is TextElement && super.equals(other);
 }
 
-enum LinkType { url, email }
+enum LinkType { url, email, phone }
```

Comment:
Which this change, I would disable phone parsing by default, since it can be quite eager with it's parsing. `linkTypes = LinkType.values` should be changed to just `url` and `email`

Reply:
i have removed phone parsing by default
please review my pr